runk/README.md

```diff
@@ -1,3 +1,9 @@
+# runk
+
+## Example: sbox demo
```

Comment:
same

Reply:
So you want me to `docker build -f Dockerfile ../..` ?

Reply:
`docker build -f ./cmd/sbox/Dockerfile .`

Reply:
that doesn't work if I put the readme in cmd/sbox like I understood you asked in https://github.com/tonistiigi/vzor/pull/2#discussion_r281045027

Reply:
it should be obvious where you are running it from the command. you can write it out if needed. Same thing for example for building dockerfile frontend in buildkit